src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
 
```

Comment:
There should be a space between `Printable` and `{`

This issue is also found in other classes implementing `Printable`